overlord/ifacestate/udevmonitor/udevmon.go

```diff
@@ -88,8 +91,15 @@ func (m *Monitor) Connect() error {
 		return fmt.Errorf("cannot start udev monitor: %s", err)
 	}
 
-	// TODO: consider passing a device filter to reduce noise from irrelevant devices.
-	m.monitorStop = m.netlinkConn.Monitor(m.netlinkEvents, m.netlinkErrors, nil)
+	var filter netlink.Matcher
+	// TODO: extend with other criteria based on the hotplug interfaces
+	filter = &netlink.RuleDefinitions{
```

Comment:
Shouldn't this be a part of a separate PR?

Reply:
Right, removed.